### DISCUSSION TITLE
How to stop creating indexes ?

### DISCUSSION THREAD

**Discussion by ducanh997** [milvus-io/milvus#28433] [Q&A and General discussion] [answered]
Milvus Version: 2.3.2

I've started creating an index using Attu, but I mistakenly configured the parameters incorrectly. Now, I want to interrupt the index creation process. However, when I use the collection.drop_index() command, the server's CPU remains high for an extended period before returning to normal. I guess index creation is still in progress in the milvus worker. My question is how to force stop the index creation process while it's ongoing?

**Comment by yhmo** [comment, marked as answer]
Milvus manages data by segments. By default, each segment size is around 512MB. Milvus builds an independent index for each segment.
After you call create_index(), the index node begins to build index for segments one by one. Then you call drop_index(), all the index tasks that in the pending list are dropped. However, the task in processing cannot be dropped until the task is finished. So, you will see the cpu remains high for a while, maybe a minute or several minutes.
